vue-app/src/api/round.ts

```diff
@@ -50,7 +53,15 @@ export async function getCurrentRound(): Promise<string | null> {
   if (fundingRoundAddress === '0x0000000000000000000000000000000000000000') {
     return null
   }
-  return fundingRoundAddress
+  const rounds = await getRounds()
+  const roundIndex = rounds.findIndex((round) =>
+    isSameAddress(round.address, fundingRoundAddress)
+  )
+
+  if (roundIndex >= Number(process.env.VUE_APP_FIRST_ROUND)) {
```

Comment:
We can default it to `0`. 

```suggestion
  if (roundIndex >= Number(process.env.VUE_APP_FIRST_ROUND || 0)) {
```

Reply:
The empty string automatically defaults to 0 but I'm good with making that more explicit 👍